src/punycode.cr

```diff
@@ -0,0 +1,168 @@
+module Punycode
```

Comment:
Not everybody knows what it is, small doc note would be nice to hint the right RFC at least :)